src/components/Signup/Signup.js

```diff
@@ -1,5 +1,232 @@
+              <div className="pw_box">
+                <div className="pw_left">
+                  <h3 className="password">* 비밀번호</h3>
+                </div>
+                <div className="pw_right">
+                  <input
+                    name="password"
+                    type="password"
+                    className="pw_input"
+                    value={inputValue.passsword}
+                    onChange={handleInput}
+                    onBlur={checkPassword}
+                    placeholder="영문, 숫자, 특수문자 포함 8자리 이상"
+                  />
+                </div>
+              </div>
+              <div className="pw_confirm_box">
+                <div className="pw_confirm_left">
+                  <h3 className="pw_confirm">* 비밀번호 확인</h3>
+                </div>
+                <div className="pw_confirm_right">
+                  <input
+                    name="confirm_password"
+                    type="password"
+                    className="pw_confirm_input"
+                    value={inputValue.confirm_password}
+                    onChange={handleInput}
+                    onBlur={confirmPassword}
+                  />
+                </div>
+              </div>
+              <div className="username_box">
+                <div className="username_left">
+                  <h3 className="username">* 이름</h3>
+                </div>
+                <div className="username_right">
+                  <input
+                    name="username"
+                    className="username_input"
+                    onChange={handleInput}
+                    onBlur={checkUsername}
+                  />
+                </div>
+              </div>
+              <div className="phone_number_box">
+                <div className="phone_number_left">
+                  <h3 className="phone_number">* 휴대폰 번호</h3>
+                </div>
+                <div className="phone_number_right">
+                  <input
+                    name="phone_number"
+                    className="phone_number_input"
+                    placeholder="- 없이 입력하세요."
+                    onChange={handleInput}
+                    onBlur={checkPhoneNumber}
+                  />
+                </div>
+              </div>
+              <div className="address_box">
+                <div className="address_left">
+                  <h3 className="address">* 주소</h3>
+                </div>
+                <div className="address_right">
+                  <input
+                    name="address"
+                    className="address_input"
+                    onChange={handleInput}
+                    onBlur={checkAddress}
+                  />
+                </div>
+              </div>
+            </form>
+          </div>
+          <div className="button">
+            <button className="cancel">취소</button>
+            <button
+              className={`${isValidForm ? 'button_activate' : 'invalid'}`}
```

Comment:
삼항연산자 잘 활용하셨네요! 
이미 {} 괄호가 있어서 그 안에 한번더 템플릿 리터럴로 감쌀 필요는 없을 것 같습니다. 
```suggestion
              className={ isValidForm ? 'button_activate' : 'invalid'}
```